clusterdock/topologies/cdh/kerberos/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM clusterdock/topology_nodebase:centos6.6
+
+RUN yum -y install krb5-server
```

Comment:
`yum install` supports installing multiple packages in one command, so consolidate all these into one call (though format it like the `Dockerfile` in the repo root where each package name is sorted on its own line for readability).